graphdatascience/graph_data_science.py

```diff
@@ -32,7 +32,31 @@ def __init__(
         auth: Optional[Tuple[str, str]] = None,
         aura_ds: bool = False,
         arrow: bool = True,
+        arrow_disable_server_verification: bool = True,
+        arrow_tls_root_certs: Optional[bytes] = None,
     ):
+        """
+        Parameters
+        ----------
+        endpoint : Union[str, Driver, QueryRunner]
+            The Neo4j endpoint to connect to
+        auth : Optional[Tuple[str, str]], default None
+            A username, password pair for database authentication.
+        aura_ds : bool, default False
+            A flag that indicates that that the client is used to connect
+            to a Neo4j Aura instance.
+        arrow : bool, default True
+            A flag that indicates that the client should use Apache Arrow
+            for data streaming if it is available on the server.
+        arrow_disable_server_verification : bool, default True
+            A flag that indicates that, if the flight client is connecting with
+            TLS, that it skips server verification. If this is enabled, all
+            other TLS settings are overridden.
```

Comment:
Is this the feature? I'm not sure what "server verification" means exactly, but it does sound odd to me to give users the ability to bypass something that sounds like security.

Reply:
It's just exposing a flag of the arrow flight client. It basically means that you can connect with tls, but the server is only accepting unencrypted, it'll still connect. It's not disabling the server-side encryption.